.github/workflows/terraform.yml

```diff
@@ -104,7 +117,24 @@ jobs:
               repo: context.repo.repo,
               body: output
             })
-      - name: terraform-apply
+      - 
+        name: terraform-apply
         id: apply
         if: inputs.apply
-        run: '[[ "$DESTROY" == "true" ]] && terraform destroy --auto-approve || terraform apply --auto-approve plan.out'
\ No newline at end of file
+        run: '[[ "$DESTROY" == "true" ]] && terraform destroy --auto-approve || terraform apply --auto-approve plan.out'
+      - 
+        name: allow aws ${{ env.AWS_USER }} user to access eks control plane
```

Comment:
ins't this part better to go with ArgoCD objects during the first scaffold execution ?